optimism/src/keccak/mod.rs

```diff
@@ -70,6 +79,77 @@ pub enum Constraint {
     IotaStateG(usize),
 }
 
+#[allow(dead_code)]
+/// The Keccak circuit
+pub(crate) type KeccakCircuit<F> = Circuit<ZKVM_KECCAK_COLS, Steps, F>;
+
+#[allow(dead_code)]
+impl<F: Field> KeccakCircuit<F> {
```

Comment:
Just some useful functions that can be called from both `main.rs` and `test.rs` to easily obtain all the required information about the keccak instances.

Reply:
In terms of design, what do you think about putting these functions into the `WitnessBuilderEnv`?